### ISSUE TITLE
Accessibility of CalnedarExtender and AutoCompleteExtender

### ISSUE THREAD

**Issue by LiTingWong** [DevExpress/AjaxControlToolkit#474]
### The installation method
- [ ] Installer
- [x] NuGet package
- [ ] A custom build from the source code

### Minimal steps to reproduce the bug
Create a calendar extender, try to access the calendar using keyboard

### Actual result
Can't access the calendar controls using keyboard.
Screen reader (JAWS) doesn't read the calendar controls since it's not accessible.

When screen reader (JAWS) is on, pressing down arrow key on an input field tied up with AutoCompleteExtender does not change the highlighted text in dropdown list. 

### Expected result
For accessibility reason, users who rely on keyboard should be able to access the controls. 

AutoCompleteExtender should still be accessible by keyboard when screen reader is on.

### Browser(s) used
Chrome, Edge, IE11

### A site deployment method
- [ ] VS development web-server,
- [x] IIS

**Comment by AlekseyMartynov** [comment]
It's unlikely that new features, such as accessibility, will be added to the Toolkit.
If you need WebForms controls with good accessibility support, I suggest that you try [DevExpress ASP.NET products](https://www.devexpress.com/products/net/controls/asp/).

- [Section 508/WCAG Compliance demo](https://demos.devexpress.com/ASPxEditorsDemos/Accessibility/Compliance.aspx)
